R/draws.R

```diff
@@ -152,6 +156,9 @@
 #' Marcel Wolbers, Alessandro Noci, Paul Delmar, Craig Gower-Page, Sean Yiu, Jonathan W. Bartlett. Reference-based
 #' imputation methods based on conditional mean imputation. \url{http://arxiv.org/abs/2109.11162}, 2021.
 #'
+#' Von Hippel, Paul T and Bartlett, Jonathan W.
+#' Maximum likelihood multiple imputation: Faster imputations and consistent standard errors without posterior draws. 2021.
+#'
```

Comment:
Do we have any document id numbers or a url ? (just thinking that we've provided number references in all of our above references ? )

Reply:
Sorry, what this the difference between this reference and the other ones? Some of them have a url, but only those ones that are in arXiv I think. I can add more details like Journal etc...?